apps/backoffice-v2/src/common/env/schema.ts

```diff
@@ -27,4 +28,6 @@ export const EnvSchema = z.object({
     value => (typeof value === 'string' ? JSON.parse(value) : value),
     z.boolean().default(true),
   ),
+  VITE_REACT_APP_PUBLIC_POSTHOG_KEY: z.string().optional(),
+  VITE_REACT_APP_PUBLIC_POSTHOG_HOST: z.string().optional(),
```

Comment:
`REACT_APP_PUBLIC` is for non-vite projects (webpack/create-react-app)
```suggestion
  VITE_POSTHOG_KEY: z.string().optional(),
  VITE_POSTHOG_HOST: z.string().optional(),
```